setup.py

```diff
@@ -30,12 +30,8 @@
 
 version = ''
 with open('requests/__init__.py', 'r') as fd:
-    reg = re.compile(r'__version__\s*=\s*[\'"]([^\'"]*)[\'"]')
-    for line in fd:
-        m = reg.match(line)
-        if m:
-            version = m.group(1)
-            break
+    version = re.search(r'__version__\s*=\s*[\'"]([^\'"]*)[\'"]',
```

Comment:
Since we're not splitting on line terminators anymore and lazily ejecting when we've found what we're looking for, the regular expression needs to be updated. We want to make sure we don't grab any cruft from a separate line that has __version__ in it.

`match` will automatically add the equivalent of `^` to the beginning of the regular expression. `search` does not. We don't want anything where there's a line that looks like

```py
# We want __version__ = "some string"
...
__version__ = "2.6.1"
```

Because we'll get the wrong result. Please fix the expression to check for line beginnings and endings.